templates/default/attachments.html

```diff
@@ -1,123 +1,100 @@
+                            {documentview id="{$container.id}-{$file.id}" type=$file.contenttype name=$file.filename
+                            url="?m=attachments&attachmentaction=viewfile&type={$attachmenttype}&id={$container.id}&fileid={$file.id}" external=true
+                            text="{$file.filename} <img src=\"img/{if preg_match('/pdf/',$file.contenttype)}pdf{elseif preg_match('/csv|excel|spreadsheet/',$file.contenttype)}xls{else}view{/if}.gif\">"}
+                            <br>
                         {/foreach}
-                        <TR class="lms-ui-button-panel">
-                            <td class="bold valign-top nobr text-right">
-                                {trans("Description:")}
-                            </td>
-                            <td class="valign-top nobr">
-                                <input type="text" name="upload[description]" size="50" value="{$upload.description}">
-                            </td>
-                            <TD class="bold nobr valign-top" colspan="2">
-                                {fileupload id="files" fileupload=$fileupload}
-                            </td>
-                            <td class="bold nobr valign-top text-right">
-                                <a href="javascript:document.upload.submit();">
-                                    <img src="img/save.gif">
-                                    {trans("Add")}
-                                </a>
-                            </td>
-                        </TR>
-                    </tbody>
-                </table>
-                </form>
-            </TD>
-        </TR>
-    </TBODY>
-</table>
-<SCRIPT type="text/javascript">
-if (getCookie('attachmentpanel-{$attachmenttype}') == '1') {
-	$('#attachmentpanel-{$attachmenttype}').show();
-}
+                    </div>
+                </div>
+            </div>
+            {foreachelse}
+                <div class="lms-ui-tab-empty-table">
+                    {trans("There are no file containers in database.")}
+                </div>
+            {/foreach}
+
+            {tab_button_panel}
+                {tab_buttons}
+                    {trans("Description:")}
+                    <input type="text" name="upload[description]" size="50" value="{$upload.description}" form="upload-form">
+                    {fileupload id="files" fileupload=$fileupload form="upload-form"}
+                    {button icon="add" onclick="javascript:document.upload.submit();" label="Add"}
+                {/tab_buttons}
+            {/tab_button_panel}
 
-$(function() {
-    $('.toggle-file-list').click(function() {
-		var containerid = parseInt($(this).attr('data-container-id'));
-		var elem = $('#files-' + containerid);
-		elem.toggle();
-		$(this).html('<img src="img/' + (elem.is(':visible')  ? 'desc' : 'asc') + '_order.gif">');
+        {/tab_table}
+    {/tab_contents}
+{/tab_container}
+
+<SCRIPT type="text/javascript">
```

Comment:
To można przerzucić do:
js/templates/attachments.js
a w tym miejscu można wstawić:
```smarty
{js}
```